Add Laravel vuln in cookie session driver

jaylinski opened the pull request:
See https://blog.laravel.com/laravel-cookie-security-releases.

I'm not sure if the 5.x branch should be marked as unfixed... :thinking: 

Also: should there be an additional entry for the `illuminate/cookie` package?

devfrey commented:
Could this be merged please? (ping @fabpot)

stof commented:
shouldn't this also register the advisory for one of the `illuminate/*` subtree split ?

stof commented:
I would say that `illuminate/cookie` is the impacted package

stof commented:
@taylorotwell do you know whether 5.x was also affected by this security issue or whether this issue was introduced as part of 6.0 ? For the advisory, it would make sense to mark 5.x as affected (without any patched version) if it is indeed the case.

jaylinski commented:
@fabpot I added an entry for `illuminate/cookie`. I also added the (unpatched) `5.5` branch, since it is still supported according to https://laravel.com/docs/7.x/releases#support-policy.

GrahamCampbell commented:
This vulnerability affects every laravel version since 4.1.0, but this PR doesn't take this into account.

GrahamCampbell commented:
> For the advisory, it would make sense to mark 5.x as affected (without any patched version) if it is indeed the case.

Yes, Laravel 4 too. I have sent in a PR for this.
